.gitattributes

```diff
@@ -1,5 +1,6 @@
 /.github export-ignore
 /.gitignore export-ignore
 /.gitattributes export-ignore
+/examples export-ignore
```

Comment:
Please use the technique I used in https://github.com/phpmyadmin/sql-parser/commit/a20131487496892d4111c1cbe0025fee35180cce because I need the files in Debian for the packaging :)